packages/modules/order/src/migrations/Migration20240604100512.ts

```diff
@@ -129,15 +129,17 @@ export class Migration20240604100512 extends Migration {
         exchange_id
     )
     WHERE exchange_id IS NOT NULL AND deleted_at IS NULL;
-    
-    
 
-    CREATE TYPE return_status_enum AS ENUM (
-        'requested',
-        'received',
-        'partially_received',
-        'canceled'
-    );
+    DO $$
```

Comment:
question: What triggered these changes?

Reply:
I had migration failing because I already had that type in the DB but since Postgres doesn't support `create type if not exist...` this is equivalent.